dynawo/sources/Models/Modelica/Dynawo/Electrical/Machines/OmegaRef/GeneratorPV.mo

```diff
@@ -28,6 +28,7 @@ model GeneratorPV "Generator with active power / frequency regulation and voltag
                               AbsorptionMax "Reactive power is fixed to its absorption limit",
                               GenerationMax "Reactive power is fixed to its generation limit");
 
+  Connectors.ImPin deltaURefPu(value(start = 0)) "Voltage regulation step in pu (base UNom)";
```

Comment:
Is that always a step ? If not, I would write "Additional voltage reference in pu (base UNom)" .